invoices/invoiceregistry.go

```diff
@@ -48,16 +48,21 @@ type HodlEvent struct {
 
 	// AcceptHeight is the original height at which the htlc was accepted.
 	AcceptHeight int32
+
+	// ResolutionResult indicates the outcome of the invoice registry update
+	// for the invoice.
+	ResolutionResult
```

Comment:
While we're here, I'm wondering about the name `HodlEvent` generally, since we return a nil event for accepts it's not really an event in the typical sense because it doesn't cover the full set events that can happen for a hodlInvoice (settle/cancel/accept). This struct is more of a `HodlResolution` since it's only populated for settles/cancels.

Thoughts on a rename of `HodlEvent` -> `HodlResolution` and then this field can just be `Outcome`/ `Result`? `HodlResolution.Outcome`, reads nicely imo. cc @joostjager ?

Reply:
Nice thinking. Yes, it definitely is a resolution. But it isn't even specific to hodling. Maybe it should just be `HtlcResolution` and `HtlcResolution.Outcome`

Reply:
Cool, I'll add another commit renaming :)
`processHodlEvent` -> `processHTLCResolution` will also be a nice rename.

Reply:
I am no fan of `HTLC`. For me it is dictionary word now and I write `Htlc` to make it not stand out so much. But that is a personal preference.

Reply:
`Htlc` in the oxford dictionary 2020 🤔